language/stdlib/modules/validator_set.mvir

```diff
@@ -130,4 +130,26 @@ module ValidatorSet {
         return;
     }
 
+  public iter_validator_addresses(i: u64): address acquires T {
+      let len: u64;
+      let validator_set: &Self.T;
+      let addr: address;
+      let peer: &Self.ValidatorInfo;
+
+      validator_set = borrow_global<T>(0x1D8);
```

Comment:
This seems a bit unfortunate as we can't return global reference and IMHO there doesn't seem to be a way round at least for now.